code/modules/mob/living/carbon/brain/robotic_brain.dm

```diff
@@ -121,6 +121,9 @@
 	to_chat(brainmob, "<b>Remember, the purpose of your existence is to serve [imprinted_master]'s every word, unless lawed  or placed into a mech in the future.</b>")
 	brainmob.mind.assigned_role = "Positronic Brain"
 
+	if(brainmob.ckey in GLOB.antag_hud_users)
+		log_and_message_admins("[key_name(brainmob)] has joined as a robot brain, after having toggled antag hud.")
```

Comment:
log_and_message_admins already uses key_name(usr) in here, you'll probably want to refactor this to just be a log and message seperately